.github/workflows/ci.yml

```diff
@@ -181,30 +194,61 @@ jobs:
         profile: minimal
         override: true
         target: ${{ matrix.target }}
+    - name: Sets OpenSSL env vars on i686
+      run: |
+        echo "OPENSSL_LIB_DIR=/usr/lib/i386-linux-gnu" >> $GITHUB_ENV
+        echo "OPENSSL_INCLUDE_DIR=/usr/include/i386-linux-gnu" >> $GITHUB_ENV
+      if: matrix.target == 'i686-unknown-linux-gnu'
+    - name: Sets OpenSSL env vars on ppc and ppc64
+      run: |
+        echo "OPENSSL_DIR=/openssl" >> $GITHUB_ENV
+      if: matrix.target != 'i686-unknown-linux-gnu'
     - name: Configure cross
       shell: bash
+      # configure and cross compile openssl locally on ppc and ppc64 to be able to run aws-smithy-client tests.
+      # since cross dropped support for openssl, we use the build script from version 0.16.
       run: |
         cat > Cross.toml << EOF
-        [build]
+        [target.i686-unknown-linux-gnu]
         pre-build = ["dpkg --add-architecture i386", "apt-get update && apt-get install --assume-yes pkg-config:i386 libssl-dev:i386"]
-        [build.env]
+        [target.i686-unknown-linux-gnu.env]
         passthrough = [
             "OPENSSL_LIB_DIR",
             "OPENSSL_INCLUDE_DIR",
         ]
+        [target.powerpc-unknown-linux-gnu]
+        pre-build = ["curl -L -s -o /tmp/openssl.sh https://github.com/cross-rs/cross/raw/c183ee37a9dc6b0e6b6a6ac9c918173137bad4ef/docker/openssl.sh && bash /tmp/openssl.sh linux-ppc powerpc-linux-gnu-"]
+        [target.powerpc-unknown-linux-gnu.env]
+        passthrough = ["OPENSSL_DIR"]
+        [target.powerpc64-unknown-linux-gnu]
+        pre-build = ["curl -L -s -o /tmp/openssl.sh https://github.com/cross-rs/cross/raw/c183ee37a9dc6b0e6b6a6ac9c918173137bad4ef/docker/openssl.sh && bash /tmp/openssl.sh linux-ppc64 powerpc64-linux-gnu-"]
+        [target.powerpc64-unknown-linux-gnu.env]
+        passthrough = ["OPENSSL_DIR"]
         EOF
-    - name: Build rust-runtime crates
+    - name: Build Smithy-rs rust-runtime crates
       uses: actions-rs/cargo@v1
       with:
         use-cross: true
         command: build
-        args: --target ${{ matrix.target }} --manifest-path "rust-runtime/Cargo.toml" --exclude aws-smithy-http-server-python --workspace ${{ matrix.non_aws_features }}
+        args: -vv --target ${{ matrix.target }} --manifest-path "rust-runtime/Cargo.toml" ${{ matrix.build_smithy_rs_exclude }} --workspace ${{ matrix.build_smithy_rs_features }}
```

Comment:
Is the `-vv` still needed?

Reply:
Without `-vv` you get pretty much no logs if a pre-build script fails. 
I'd lean towards leaving it in - I only check CI logs when it fails and there I do appreciate having _something_ to look at.